src/context/ChartPropertiesSchema.ts

```diff
@@ -96,19 +96,50 @@ const xAxisSection: ChartPropertySchemaSection = {
       options: ["normal", "tozero", "nonnegative"],
       defaultValue: "normal",
     },
-    {
-      name: "xTickLabelMaxLength",
-      displayName: "X axis tick label max. length",
-      type: "text",
-      defaultValue: 40,
-    },
     {
       name: "xAxisTickAngle",
-      displayName: "X axis label rotation",
+      displayName: "Label rotation",
       type: "radio",
       options: ["0", "45", "-45", "-90"],
       defaultValue: "0",
     },
+    {
+      name: "xAxisTickMode",
+      displayName: "Tick label mode",
+      type: "radio",
+      options: ["auto", "manual"],
+      defaultValue: "auto",
+    },
+    {
+      name: "xAxisTickLabelLength",
+      displayName: "Tick label max. length",
+      type: "text",
+      defaultValue: 40,
+    },
+    {
+      name: "xAxisTickInterval",
+      displayName: "Tick interval",
+      type: "text",
+      defaultValue: 1,
+    },
+    {
+      name: "xAxisFirstTickLabel",
+      displayName: "First tick label",
+      type: "text",
+      defaultValue: "",
+    },
+    {
+      name: "xAxisLastTickLabel",
+      displayName: "Last tick label",
+      type: "text",
+      defaultValue: "",
+    },
+    {
+      name: "xAxisNTicks",
+      displayName: "Show n ticks",
```

Comment:
Small nitpick, uppercase `N` might read better!

Reply:
I did google this whilst implementing it and couldn't find conclusive support one way or the other! Tweaked to uppercase `N` cheers.